test/test_debugger.cpp

```diff
@@ -1144,9 +1153,10 @@ TEST_SUITE("debugger")
     {
         start_kernel();
         timer t;
-        xeus::xcontext context;
+        auto context_ptr = xeus::make_zmq_context();
         {
-            debugger_client deb(context, KERNEL_JSON, "debugger_init.log");
+            debugger_client deb(*context_ptr, KERNEL_JSON, "debugger_init.log");
+            deb.start();
```

Comment:
This should start the debugger_client (or rather make a call to the client from xeus-zmq to start)